I20231219-1800 - BUILD FAILED, I20231220-0500 unstable

See https://ci.eclipse.org/releng/job/Builds/job/I-build-4.31/50/console

```
01:18:15  [apitooling.analysis] ==========================
01:18:15  [apitooling.analysis] API tools verification check : 33631ms
01:18:15  
01:18:15  BUILD FAILED
01:18:15  /home/jenkins/agent/workspace/Builds/I-build-4.31/eclipse.platform.releng.aggregator/eclipse.platform.releng.aggregator/cje-production/gitCache/eclipse.platform.releng.aggregator/eclipse.platform.releng.tychoeclipsebuilder/eclipse/buildScripts/api-tools-builder.xml:115: java.lang.ArrayIndexOutOfBoundsException
01:18:15  
01:18:15  Total time: 50 seconds
01:18:16  An error has occurred. See the log file
01:18:16  /home/jenkins/agent/workspace/Builds/I-build-4.31/eclipse.platform.releng.aggregator/eclipse.platform.releng.aggregator/cje-production/tmp/workspace-apitoolingsLogs/.metadata/.log.
01:18:16  + [[ 13 -ne 0 ]]
01:18:16  + echo 'Failed in Generate API tools reports stage'
01:18:16  Failed in Generate API tools reports stage
01:18:16  + exit 1
```

--- a/bundles/org.eclipse.swt.cocoa.macosx.aarch64/forceQualifierUpdate.txt
+++ b/bundles/org.eclipse.swt.cocoa.macosx.aarch64/forceQualifierUpdate.txt
@@ -1,3 +1,4 @@
 # To force a version qualifier update add the bug here
 Bug 527899 [9] Implement JEP 280: Indify String Concatenation
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1659
+https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1682

--- a/bundles/org.eclipse.swt.cocoa.macosx.x86_64/forceQualifierUpdate.txt
+++ b/bundles/org.eclipse.swt.cocoa.macosx.x86_64/forceQualifierUpdate.txt
@@ -3,3 +3,4 @@ Bug 434039 - SWT fragment source is not always included in SDK or repository
 Bug 436000 - Comparator errors in I20140527-2000 build
 Bug 527899 [9] Implement JEP 280: Indify String Concatenation
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1659
+https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1682

--- a/bundles/org.eclipse.swt.gtk.linux.aarch64/forceQualifierUpdate.txt
+++ b/bundles/org.eclipse.swt.gtk.linux.aarch64/forceQualifierUpdate.txt
@@ -2,3 +2,4 @@
 Bug 527899 [9] Implement JEP 280: Indify String Concatenation
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1608
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1659
+https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1682

--- a/bundles/org.eclipse.swt.gtk.linux.ppc64le/forceQualifierUpdate.txt
+++ b/bundles/org.eclipse.swt.gtk.linux.ppc64le/forceQualifierUpdate.txt
@@ -3,3 +3,4 @@ Bug 434039 - SWT fragment source is not always included in SDK or repository
 Bug 527899 [9] Implement JEP 280: Indify String Concatenation
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1608
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1659
+https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1682

--- a/bundles/org.eclipse.swt.gtk.linux.x86_64/forceQualifierUpdate.txt
+++ b/bundles/org.eclipse.swt.gtk.linux.x86_64/forceQualifierUpdate.txt
@@ -3,3 +3,4 @@ Bug 434039 - SWT fragment source is not always included in SDK or repository
 Bug 527899 [9] Implement JEP 280: Indify String Concatenation
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1608
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1659
+https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1682

--- a/bundles/org.eclipse.swt.win32.win32.x86_64/forceQualifierUpdate.txt
+++ b/bundles/org.eclipse.swt.win32.win32.x86_64/forceQualifierUpdate.txt
@@ -3,3 +3,4 @@ Bug 434039 - SWT fragment source is not always included in SDK or repository
 Bug 436000 - Comparator errors in I20140527-2000 build
 Bug 527899 [9] Implement JEP 280: Indify String Concatenation
 https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1659
+https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/issues/1682

